web/gds-admin-ui/src/layouts/LeftSidebar.js

```diff
@@ -7,7 +6,7 @@ import { getMenuItems } from 'helpers/menu';
 import { getDirectoryLogo, getDirectoryName, getDirectoryURL } from 'utils';
 
 
-const SideBarContent = ({ hideUserProfile }: SideBarContentProps) => {
+const SideBarContent = ({ hideUserProfile }) => {
```

Comment:
Is this `hideUserProfile` something to verify too?

I saw code similar to this 🔽  was modified in other files, just pin here to check and see if we are missing this one. 
```
type LeftSidebarProps = {
    hideLogo: boolean,
    hideUserProfile: boolean,
    isLight: boolean,
    isCondensed: boolean,
};
```

Reply:
This verification no really matter